docker/ci/install/install-rocksdb

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# RocksDB dependencies
+# Install gflags
```

Comment:
I'll try running it without them, but it lists gflags and zstandard as dependencies for CentOS (https://github.com/facebook/rocksdb/blob/master/INSTALL.md)